docs/en/setup/service-agent/java-agent/How-to-enable-kafka-reporter.md

```diff
@@ -12,8 +12,10 @@ collector.backend_service=${SW_AGENT_COLLECTOR_BACKEND_SERVICES:127.0.0.1:11800}
 
 # Kafka producer configuration
```

Comment:
One missing change

> In other words, the following configure cannot be omitted.

These words should be updated. All configurations have been activated in the `agent.config`.